src/psyclone/tests/domain/lfric/lfric_builtins_test.py

```diff
@@ -250,6 +250,34 @@ def test_builtin_operator_arg():
             in str(excinfo.value))
 
 
+def test_builtin_invalid_data_type():
+    ''' Check that we raise appropriate error if we encounter a
+    built-in that takes something other than an argument of a 'real'
+    or an 'integer' data type. '''
+    old_name = lfric_builtins.BUILTIN_DEFINITIONS_FILE[:]
```

Comment:
Please could you change this to use monkeypatch in order to modify the object.

Reply:
Heh, as soon as I put this test in I knew you'd ask me to monkeypatch it but I was lazy :smile: Done.